substrate/frame/paged-list/src/paged_list.rs

```diff
@@ -26,13 +26,13 @@
 use alloc::vec::Vec;
 use codec::{Decode, Encode, EncodeLike, FullCodec};
 use core::marker::PhantomData;
-use frame_support::{
-	defensive,
-	storage::StoragePrefixedContainer,
-	traits::{Get, StorageInstance},
-	CloneNoBound, DebugNoBound, DefaultNoBound, EqNoBound, PartialEqNoBound,
+
+use frame::{
+	deps::{frame_support, sp_io},
```

Comment:
Usage of `deps` is generally discouraged, are they absolutely necessary in this file?

Reply:
This is an issue regarding the defensive! macro in frame_support that actually has a dependency with frame_support
I couldnt find a way to fix it without having to rewrite the macro 
https://paritytech.github.io/polkadot-sdk/master/src/frame_support/traits/misc.rs.html#67-95

Reply:
for the other i will add it to the prelude